packit.fmf

```diff
@@ -0,0 +1,5 @@
+discover:
+  how: fmf
+  url: https://src.fedoraproject.org/tests/scap-security-guide.git
```

Comment:
What about adding `filter: 'tag: Packit'` so we would need to approve which tests are selected for gating by adding `Packit` tag to them?

Reply:
I agree. Filter is there and I have also updated smoke-test with the new `Packit` tag